.github/workflows/release.yml

```diff
@@ -0,0 +1,82 @@
+name: Release
```

Comment:
We need to merge the release related steps from `workflwos/build.yml` to this `release.yml`. So delete the repeated steps from the `build.yml` and add them to this file. Also check if there is any release step missing in this file.